common/src/main/kotlin/io/homeassistant/companion/android/common/data/servers/SecurityInfo.kt

```diff
@@ -0,0 +1,4 @@
+package io.homeassistant.companion.android.common.data.servers
+
+// This should never be persisted and we should always check the info before making a call
+class SecurityInfo(val isOnInternalNetwork: Boolean, val hasHomeSetup: Boolean, val locationEnabled: Boolean)
```

Comment:
This is a state, why not call it SecurityState so it is more obvious it shouldn't be persisted?